README.md

```diff
@@ -73,6 +73,33 @@ npm install .
 
 Open `vscode-hie-server/` in Visual Studio Code and press `F5` to open a new window with the extension loaded.
 
+### Hoogle Docs on hover
+
+HIE supports fetching docs from hoogle on hover. It can use the Hoogle database for 
+your user, or a project specific database.
+
+To generate a global hoogle database for your user, run
+
+```bash
+$ hoogle generate
+```
+
+To generate a project specific database for a stack project, run
+
+```bash
+$ stack hoogle
```

Comment:
It seems you have to do a `stack build` after this, otherwise `hie` can't load files in the project, the .stack-work has the pre-haddock build artefacts listed, which ghc-mod uses